### ISSUE TITLE
Couldn't get an API Key

### ISSUE THREAD

**Issue by oywino** [marthoc/docker-deconz#191]
All SW and firmware up to date. deCONZ runs fine and all ZigBee devices are visible and controllable.
But when I try to establish integration from Home Assistant (v0.103), all I get is the "Couldn't get an API Key" during the 60 sec Authenticate App from deCONZ.
I have tried adding the following to my _configuration.yaml_ ;
```
deconz:
  host: 192.168.0.9
  port: 80
```
But I do not know if the above is needed, or if the IP and port is correct. What I do know is that this is the IP where deCONZ is running and that's where I start Phoscon.
Any additional help is appreciated.

**Comment by marthoc** [comment]
You may want to create an issue on the Home Assistant GitHub as this doesn't sound like an issue with the container. @Kane610 any ideas?

**Comment by oywino** [comment]
I managed to solve the issue by completely factory reset the ConBee II and move it to a local Windows computer and set it up again from scratch. Apparently, at some point during first time setup from the container version of deCONZ, the ConBee got stuck with an invalid IP addresse out-of-range (wrong sub-net) that kept popping up again no matter where I moved the stick around - until I factory reset it. 
Now everything works just fine.
